tracing-appender/src/rolling.rs

```diff
@@ -4,12 +4,55 @@ use std::fmt::Debug;
 use std::io;
 use std::path::Path;
 
+/// A file appender with the ability to rotate log files at a fixed schedule.
+///
+/// `RollingFileAppender` implements [`std:io::Write` trait][write] and will block on write operations.
```

Comment:
it would be good to also have a way to use this without [`NonBlocking`]...would need a makewriter impl. but, that needs #675